website/docs/components/table/partials/code/component-api.md

```diff
@@ -1,49 +1,107 @@
   </C.Property>
-  <C.Property @name="isFixed" @type="boolean" @values={{array "false" "true" }} @default="false">
+  <C.Property @name="isFixedLayout" @type="boolean" @values={{array "false" "true" }} @default="false">
     If set to `true`, the `table-display`(CSS) property will be set to `fixed`, which will automatically distribute columns equally based on the total width of the table.
   </C.Property>
   <C.Property @name="density" @type="enum" @values={{array "short" "medium" "tall" }} @default="medium">
     If set, determines the density, or height, of the table’s rows.
   </C.Property>
+  <C.Property @name="align" @type="enum" @values={{array "left" "center" "right" }} @default="left">
+    If set, determines the text alignment for all cell (`td`) content in a table.
+  </C.Property>
   <C.Property @name="valign" @type="enum" @values={{array "top" "middle" }} @default="top">
     If set, determines the vertical alignment for all cell (`td`) content in a table. Does not apply to table headers (`th`).
   </C.Property>
   <C.Property @name="caption" @type="string">
-    Adds a (non-visible) caption for users with assistive technology. If set on a sortable table, the provided table caption is paired with the automatically generated sorted message text.
+    Adds a (non-visible) caption for users with assistive technology. If set on a sortable table, the provided table caption is paired with the automatically generated sorted message text. Note: it is best practice to provide a table caption for users with assistive technology.
   </C.Property>
   <C.Property @name="...attributes">
     Supported for the `Hds::Table` component.
   </C.Property>
+  <C.Property @name="onSort" @type="function">
+    Automatically applied to sortable table headers, `onSort` is the callback function that is invoked when one of the sortable table headers is clicked (or has a keyboard interaction performed). The function receives the values of `sortBy` and `sortOrder` as arguments.
+  </C.Property>
+</Doc::ComponentApi>
+
+### Table::Tr
+
+The `Hds::Table::Tr` component is a template-only component.
+
+This component is not eligible to receive interactions (e.g., it cannot have an `onClick` event handler attached directly to it). Instead, an interactive element should be placed _inside_ of a `th` or `td` element.
+
+It can contain `Hds::Table::Th` or `Hds::Table::Td` components.
+
+<Doc::ComponentApi as |C|>
+    <C.Property @name="...attributes">
+    This component supports the use of `...attributes`.
+  </C.Property>
+</Doc::ComponentApi>
+
+### Table::Th
+
+This component is not eligible to receive interactions (e.g., it cannot have an `onClick` event handler attached directly to it). Instead, an interactive element should be placed _inside_ of a `th` or `td` element.
+
+If a `th` is passed as the first "cell" of a table body row, it has `scope="row"` automatically applied to it for accessibility purposes.
+
+<Doc::ComponentApi as |C|>
```

Comment:
These arguments are missing:
- `@width`
- `@scope`

Reply:
They should not be using scope at all, that's why it's not there. I'll add width.

Reply:
> They should not be using scope at all

When they build a table by themselves, just using the building blocks, are they not expected to use the `@scope` for the `Th`? If they are, then we should expose the argument and explain how/when to use it. This is more in line with our principle _"Guidance over control"_.

Reply:
Sorry, what I'm saying is, we automatically apply it in the appropriate way. If the `Th` element is inside of the `tbody`, we add `scope="row"`, since that's the only possible combination where this application would be correct. That way, the accessibility is built-in and the consumer doesn't have to worry about that detail.

Reply:
Also, I can add it to the documentation, it seemed like we haven't been adding things that are either private or things the consumer doesn't have to worry about. So I could go either way, WDYT?

Reply:
It's a remote case, but if consumers do something like this:
```
<Hds::Table>
  ...
  <:body>
    <Hds::Table::Tr>
      <Hds::Table::Th>...</Hds::Table::Th>
      <Hds::Table::Td>...</Hds::Table::Td>
      <Hds::Table::Td>...</Hds::Table::Td>
    </Hds::Table::Tr>
  </:body>
</Hds::Table>
```
It will not be applied. So in that sense, we should be thorough and document it (with a note that if they use contextual subcomponents, it will be applied for them automatically)

But if you think is too much, no big deal, we can leave this specific prop out.

Reply:
oh, good callout. I think I should add it then, since a lot of these sort of fall in this same category (e.g., if you use a custom invocation then you'll have to define a lot of things yourself) 👍